pom.xml

```diff
@@ -571,38 +571,38 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-enforcer-plugin</artifactId>
-                    <version>3.0.0-M2</version>
+                    <version>3.4.1</version>
```

Comment:
@wburns I'm not familiar, but would be fine to add a dependabot from now on for such updates?

Reply:
💯

Reply:
created https://github.com/Hyperfoil/Hyperfoil/issues/348 to keep track of this